### PR TITLE
Fe605 my books delete

### PR DESCRIPTION
**Pull request by MatthewSpeaks**
Added MyLibrary component for user dashboard. Simple table that allows a user to view their books, select, and delete them from the database. 

Future update to include: 
- once Customers are setup and the backend controller is updated, this should dynamically update to display that specific users books. 
- toggle functionality to toggle availability of the books without having to delete. Will need to finish handleToggle method and build a Book Controller method on the back end to receive those update requests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@emotion/react": "^11.11.4",
-    "@emotion/styled": "^11.11.0",
```
Comment (dalpiad) [inline package.json:14]: I don't know why or how but something in your code breaks the home page visually (and doesn't allow you to easily access the drop down).  I assume it's something either to do with the "@emotion/styled" going from version 11.11.0 to 11.11.5 or the CSS file but neither seems to directly change the page so I'm not sure what going on.

**Review by dalpiad — CHANGES_REQUESTED** [review]
Bottom-line: the home page doesn't render correctly on this branch.  I don't know why or how but something in your code breaks the home page visually (and doesn't allow you to easily access the drop down).  I assume it's something either to do with the "@emotion/styled" going from version 11.11.0 to 11.11.5 or the CSS file but neither seems to directly change the page so I'm not sure what going on.  If you have the the app running and you are on the home page, then switch back and forth from "frontend-dev" to your branch, the page breaks, and upon switching back it is fixed again.

File: package.json (hunk 1 above)
Comment (MatthewSpeaks) [reply package.json:14]: This was some CSS styling I was trying for the isAvailable toggle, but ended up deleting it. Not sure why this is breaking something however as it's a dependency that isn't being used. 

I went in and deleted it.

**Comment by MatthewSpeaks** [comment]
resubmitting pull request after deleting package.json dependency.

**Pull request closed without merging**